src/modules/translation/languages/en.json

```diff
@@ -951,7 +951,8 @@
   "mint_items_modal": {
     "title": "Mint Items",
     "add_item": "Add Item",
-    "no_items": "You have no items available to mint for \"{name}\""
+    "no_items": "You have no items available to mint for \"{name}\"",
+    "mint": "Mint"
```

Comment:
What do you think of moving `mint` to global?

Reply:
will do